src/sentry/models/user.py

```diff
@@ -98,6 +102,16 @@ class User(BaseModel, AbstractBaseUser):
             "active. Unselect this instead of deleting accounts."
         ),
     )
+    is_unclaimed = models.BooleanField(
+        _("unclaimed"),
+        default=False,
+        help_text=_(
+            "Designates that this user was imported via the relocation tool, but has not yet been "
+            "claimed by the owner of the associated email. Users in this state have randomized "
+            "passwords - when email owners claim the account, they are prompted to reset their "
+            "password and do a one-time update to their username."
+        ),
+    )
```

Comment:
If we relocate a user that has tokens will the relocated tokens work whilst the user is unclaimed?

Reply:
Do you mean session tokens, or are there other kinds of tokens you had in mind? The `session.Sessions` Django model is not included in exports, so I believe that should be sufficient to prevent session carryover.

Reply:
Not sessions, apitoken is a user bound API token that gives access to all the orgs a user would. I remember that earlier the plan would be to create new apitokens for users that were migrated preserving the shape of the token but not the token key value. 

If we move a user and create it as unclaimed, would the reconstructed tokens be usable before the user is claimed?

Reply:
ApiToken is `Global` scope, so it will never be imported into SaaS: https://github.com/getsentry/sentry/blob/24ec43b80d47999383bf7711a2e1507e5df223fe/src/sentry/models/apitoken.py#L34

It only gets included in the export for self-hosted backup/restore purposes.